docs/blog/2024-january-update.rst

```diff
@@ -0,0 +1,107 @@
+.. post:: Jan 11, 2023
+   :tags: stats, year in review, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Update - January 2024
+===============================================
+
+Hey folks, Eric Holscher here, one of the co-founders of Write the Docs.
+I'm here with our traditional January community update to give the newsletter team the month off.
+
+2024 is looking to be a refinement of our approach to 2023,
+with a few new experiments thrown in.
+
+Our 2024 conferences
+--------------------
+
+It was lovely to see everyone again in Portland & Australia conferences this year,
+and to continue connecting virtually with our Atlantic conference.
+We're planning for 2024 to be similar to last year:
+
+* `Portland <https://www.writethedocs.org/conf/portland/2024/>`_ will be in-person on **April 14-16**.
+* `Atlantic <https://www.writethedocs.org/conf/atlantic/2024/>`_ will be a virtual event, spanning the US and European timezones, on **September 22-24**.
+* Australia will likely be in-person again in December, but we're still working on the details.
+
+We are currently accepting `talk proposals <https://www.writethedocs.org/conf/portland/2024/cfp/>`_ and selling tickets for our Portland conference.
+**Submit your talk proposals by January 15th**.
+
+We hope that you'll work with us to make all of our events unique again this year by submitting talks and showing up in-person or virtually.
+
+Community board and teams
+-------------------------
+
+We've had our Community Board & CoreOps team in place for a year now,
+and we've been posting `quarterly updates <https://www.writethedocs.com/blog/archive/tag/community-board/>`_ for everyone to read.
+We continue to refine our processes and approach,
+and would love any feedback that folks have.
+
+Overall, we're pretty happy with the structure that we have in place,
+and are looking forward to continuing to refine it in 2024.
+
+Salary Survey
+-------------
+
+We just finished up the 2023 Salary Survey, and we're excited that we had **over 1,000** responses.
+There were enough contractors this year for us to break out some high-level data,
+along with our normal breakdown of full-time employees.
+
+We plan to release the results in Q2 of the year,
```

Comment:
```suggestion
We plan to release the results in Q2, in time for the Portland conference, 
```

Reply:
Pending confirmation from Kay that it's possible